openldap/assets/dashboards/openldap_overview.json

```
@@ -0,0 +1,584 @@
+{
+  "title": "OpenLDAP - Overview",
+  "description": "This dashboard provides an overview of OpenLDAP metrics from the `cn=Monitor` backend of your OpenLDAP servers. \n\nFor more information, see:\n\n- Our [official OpenLDAP documentation](https://docs.datadoghq.com/integrations/openldap/)",
+  "widgets": [
+    {
+      "id": 7222283655321854,
+      "definition": {
+        "type": "note",
+        "content": "## About OpenLDAP\nThis dashboard shows you the status of your OpenLDAP server, as well as its connections, operations, threads, and logs to help you monitor and maintain the health of your database.\n\n##### [Official Datadog OpenLDAP documentation ↗](https://docs.datadoghq.com/integrations/openldap/)\n\nYou can clone this dashboard and modify its contents or copy and paste widgets to create your own visualizations for your OpenLDAP server",
+        "background_color": "white",
+        "font_size": "14",
+        "text_align": "left",
+        "vertical_align": "top",
+        "show_tick": false,
+        "tick_pos": "50%",
+        "tick_edge": "left",
+        "has_padding": true
+      },
+      "layout": {
+        "x": 0,
+        "y": 0,
+        "width": 6,
+        "height": 4
+      }
+    },
+    {
+      "id": 1,
+      "definition": {
+        "title": "Overview",
+        "title_align": "center",
+        "type": "group",
+        "show_title": true,
+        "layout_type": "ordered",
+        "widgets": [
+          {
+            "id": 5794216397682308,
+            "definition": {
+              "title": "OpenLDAP Service Check",
+              "title_size": "16",
+              "title_align": "left",
+              "time": {
+                "live_span": "10m"
+              },
+              "type": "check_status",
+              "check": "openldap.can_connect",
+              "grouping": "cluster",
+              "group_by": [
+                "host",
+                "url"
+              ],
+              "tags": [
+                "$host",
+                "$url"
+              ]
+            },
+            "layout": {
+              "x": 0,
+              "y": 0,
+              "width": 2,
+              "height": 2
+            }
+          },
+          {
+            "id": 936573064721974,
+            "definition": {
+              "title": "Active Connections",
+              "title_size": "16",
+              "title_align": "left",
+              "time": {
+                "live_span": "15m"
+              },
+              "type": "query_value",
+              "requests": [
+                {
+                  "q": "max:openldap.connections.current{$host,$url}",
+                  "aggregator": "max"
```

```suggestion
                  "q": "sum:openldap.connections.current{$host,$url}",
                  "aggregator": "last"
```

This means that it will sum open connections by host and url using the last reported value. Seems to be what we would want when someone is trying to know how many `active connections` currently exist.